models/netsuite2/intermediate/base/int_netsuite2__transaction_lines.sql

```diff
@@ -28,7 +33,9 @@ joined as (
     left join transaction_accounting_lines
         on transaction_lines.transaction_line_id = transaction_accounting_lines.transaction_line_id
         and transaction_lines.transaction_id = transaction_accounting_lines.transaction_id
+
+    where transaction_accounting_lines.accounting_book_id in (select accounting_book_id from accounting_books where is_primary)
```

Comment:
I am a bit worried to directly apply this filter in this model. Could it still be worthwhile to include non primary accounting book accounts in downstream calculations? I wonder if it would be more appropriate to simply add the accounting book fields to these models and allow the end users to filter appropriately based on their needs. Thoughts?

Reply:
I agree with you @fivetran-joemarkiewicz. `accounting_book_id` is an important dimension in the transaction line data. This filter could be applied downstream in a production view if needed.